physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -810,6 +869,30 @@
   type = real
   kind = kind_phys
   intent = inout
+[peak_hr_out]
+ standard_name = peak_hr_fire
```

Comment:
How about `time_of_peak_fire_emissions` and then have `h` as the unit so that we know it is hours.

Reply:
@grantfirl It will be updated to use time_of_peak_fire_emissions as standard name. We multiplied 3600 in the code like (peak_hr(i,j)= 23.0* 3600), and will use "s" as the unit. Thanks.